meson.build

```diff
@@ -0,0 +1,40 @@
+project('provisioningd', 'cpp',default_options: ['c_std=c18','cpp_std=c++23','cpp_args=-Wno-subobject-linkage'])
```

Comment:
why do we need 'cpp_args=-Wno-subobject-linkage' ? 
I dont think we need to disable warnings

Reply:
please set 'buildtype=debugoptimized',